packages/rum/src/trackPageActivities.ts

```diff
@@ -1,11 +1,11 @@
 import { monitor, Observable } from '@datadog/browser-core'
 import { LifeCycle, LifeCycleEventType, Subscription } from './lifeCycle'
 
-// Delay to wait for a page activity to validate the tracking process
+// Delay to wait for a page activity to validate the tracking process in ms
 export const PAGE_ACTIVITY_VALIDATION_DELAY = 100
-// Delay to wait after a page activity to end the tracking process
+// Delay to wait after a page activity to end the tracking process in ms
 export const PAGE_ACTIVITY_END_DELAY = 100
-// Maximum duration of the tracking process
+// Maximum duration of the tracking process in ms
```

Comment:
It clarifies the unit of the constant as we have some outputs in ns and some in ms and helps to see if the variables are homogeneous in the durations calculation.

Reply:
OK IMO it is maybe not the right approach since all the constants should be in ms and time usage in the code base should be in ms.
Conversion to ns should be done as late as possible because it is only needed for the intake format.

Reply:
so?

Reply:
so I think it is still relevant to specify that theses constants are in ms as it gives more context and helps understanding their purpose. And you are right, as ms is the standard, I will keep the `msToNs` outside of the tracking timing logic.

Reply:
I am not against it but if we do that here we should also do it for every other time constants in the code base

Reply:
ok, out of the scope of this PR then